wordpress-mobile/WordPress-iOS
# Fix status bar override for app store screenshots

Fixes #15265

We were previously using `SimulatorStatusMagic` to control the status bar in app store screenshots. We're now using the `override_status_bar` setting in Fastlane to control the status bar, and our old setup was breaking the status bar overrides. This removes `SimulatorStatusMagic` since it's no longer needed.

### Screenshots

Notice the status bar changes in the before/after screenshots below:

Before|After
-|-
<img width="1366" alt="iPad Pro (12 9-inch) (2nd generation)-1-dark-Editor" src="https://user-images.githubusercontent.com/8658164/98711837-6c500600-237d-11eb-9e45-1053ec7ec880.png">|<img width="1366" alt="iPad Pro (12 9-inch) (2nd generation)-1-dark-Editor" src="https://user-images.githubusercontent.com/8658164/99283893-82ead700-282d-11eb-8fea-95bb9509f43c.png">


### To test

1. Navigate to the fastlane directory: `cd Scripts/fastlane`
2. Generate screenshots for at least one locale: `bundle exec fastlane screenshots language:"en-US"`
3. Confirm the status bar shows the expected details: date & time `9:41 AM Tue Jan 9`, WiFi active, full cellular connection, full battery

### Release

PR submission checklist:
